packages/nimble-components/CONTRIBUTING.md

```diff
@@ -77,15 +77,34 @@ Before building a new component, 3 specification documents need to be created:
 
 ## Develop new components
 
-### Marking a component as in development
+### Marking a component as experimental / in-development / incubating
```

Comment:
nit, avoid very long headers with punctuation as it results in poorer links (notice in the rendered view the extra hyphen characters and extra verbosity).

```
CONTRIBUTING.md#marking-a-component-as-experimental--in-development--incubating
```

Also not a super strong convention at all but if you don't need to change the header text it can avoid breaking existing anchors on a page (not as a big a deal as breaking a page link altogether).

I'd recommend not changing the header in this case. I'm also not sure the hyphen on in-development is actually needed? I thought it's used to disambiguate like `in-development component` vs `in development-component` but here there isn't a need to disambiguate. I also don't know grammar so I'll take your word for it :P